postgrescontroller/pkg/stub/handler.go

```diff
@@ -0,0 +1,587 @@
+package stub
+
+import (
+	"os"
+	"io/ioutil"
+	"context"
+	"strconv"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/demo/postgrescontroller/pkg/apis/postgrescontroller/v1"
+
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	apiv1 "k8s.io/api/core/v1"
+	apiutil "k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"strings"
+	"fmt"
+	"time"
+
+	_ "github.com/lib/pq"
+
+	postgresv1 "github.com/demo/postgrescontroller/pkg/apis/postgrescontroller/v1"
+
+	"database/sql"
+)
+
+const controllerAgentName = "postgres-controller"
+
+const (
+	// SuccessSynced is used as part of the Event 'reason' when a Foo is synced
+	SuccessSynced = "Synced"
+	// ErrResourceExists is used as part of the Event 'reason' when a Foo fails
+	// to sync due to a Deployment of the same name already existing.
+	ErrResourceExists = "ErrResourceExists"
+
+	// MessageResourceExists is the message used for Events when a resource
+	// fails to sync due to a Deployment already existing
+	MessageResourceExists = "Resource %q already exists and is not managed by Foo"
+	// MessageResourceSynced is the message used for an Event fired when a Foo
+	// is synced successfully
+	MessageResourceSynced = "Foo synced successfully"
+)
+
+const (
+      PGPASSWORD = "mysecretpassword"
+      MINIKUBE_IP = "192.168.99.100"
```

Comment:
Change this to set the MINIKUBE_IP from environment variable.

Check 

https://github.com/cloud-ark/kubeplus/blob/master/postgres-crd-v2/controller.go#L73

You will have to modify the deployment yaml file to set this environment variable.
For that check here:
https://github.com/cloud-ark/kubeplus/blob/master/postgres-crd-v2/artifacts/deployment/deployment-minikube.yaml#L22